tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_widgets_Shell.java

```diff
@@ -727,7 +727,7 @@ public void a_test_setRegion() {
 	Shell shell2 = new Shell(display, SWT.NO_TRIM);
 	assertNull(":d:", shell2.getRegion());
 	shell2.setRegion(region);
-	assertEquals(":e:", region.handle, shell2.getRegion().handle);
+	assertTrue(":e:", region.equals(shell2.getRegion()));
```

Comment:
Why not just use `assertEquals` for the regions?

Reply:
Yes right haha